dts/bindings/sensor/st,lis2dw12-common.yaml

```diff
@@ -104,6 +104,39 @@ properties:
       - 3
       - 4
 
+  fifo-mode:
+    type: int
+    default: 0
+    description: |
+      Specify the sensor FIFO mode. Default is 0
```

Comment:
Please document in the description why this default value was chosen (is it a hardware register reset value?), not repeat what the value is

Reply:
Done :)